### DISCUSSION TITLE
Alist挂在阿里云盘出现failed get link: invalid X-Device-Id

### DISCUSSION THREAD

**Discussion by JiaLeDL** [AlistGo/alist#3381] [Q&A]
突然打开挂载的阿里云盘文件打开出现failed get link: invalid X-Device-Id
其他网盘的正常使用

**Comment by crowhidden** [comment]
一样的问题

**Reply by JiaLeDL** [reply]
昨天晚上还好好的 今天打开就突然这样了 把云盘删了重新挂载也是这样

**Reply by crowhidden** [reply]
阿里网盘更新了api验证

**Reply by JiaLeDL** [reply]
等以后能更新恢复使用吗

**Reply by crowhidden** [reply]
官网有内测链接，可以去看看，填个问卷表

**Reply by JiaLeDL** [reply]
有链接吗 没找到😭

**Comment by 14960** [comment]
阿里云官方对第三方客户端和工具的视频下载、观看进行屏蔽了

**Reply by JiaLeDL** [reply]
那现在有啥解决方法吗  还是说之后就都不能用了

**Reply by 14960** [reply]
不知道更新 Alist 能不能绕过这个屏蔽，等待Alist更新看看

**Comment by erchezhang** [comment]
+1

**Comment by JayHawSG** [comment]
+1

**Comment by brostart** [comment]
我挂载的别人的alist盘里的视频可以播放，自己的阿里云盘就出现failed get link: invalid X-Device-Id

**Reply by fibbr** [reply]
一样的情况- -

**Comment by BachWV** [comment]
可以挂载分享链接

**Reply by starsink-d** [reply]
怎么搞？

**Reply by jSKViN** [reply]
> 怎么搞？

https://alist.nn.ci/zh/guide/drivers/aliyundrive.html#root-folder-file-id

**Comment by JoeChen2me** [comment]
+1  今晚想看剧发现连不上了

**Comment by starsink-d** [comment]
非常难受，我挂载的好多东西都下载不了啦




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年2月13日(星期一) 晚上9:55
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [alist-org/alist] Alist挂在阿里云盘出现failed get link: invalid X-Device-Id (Discussion #3381)





 
+1  今晚想看剧发现连不上了

**Reply by JoeChen2me** [reply]
要是后续解决不了，就直接弃坑了

**Comment by starsink-d** [comment]
是的，解决不了我就全换成123云盘了，多整几个手机号，速度还快




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年2月13日(星期一) 晚上9:58
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [alist-org/alist] Alist挂在阿里云盘出现failed get link: invalid X-Device-Id (Discussion #3381)





 
要是后续解决不了，就直接弃坑了

**Reply by 14960** [reply]
兄弟，123盘你能正常看视频么？我看视频好卡，无论是直接浏览器看还是播放器挂WEBDAV都一样，相当于完全看不了。但是下载东西的话能满速。

**Comment by defpis** [comment]
希望大佬解决一下，刚刚入坑alist，结果阿里云盘用不了

**Comment by nb-sb** [comment]
+1

**Comment by Daniel041** [comment]
现在解决了吗，我刚试的还是不行

**Comment by lingok1** [comment]
+1

**Comment by 6hzzz** [comment]
https://www.52pojie.cn/thread-1745677-1-1.html

**Comment by Huanxige52** [comment]
+1 +1  同样的问题

**Reply by Huanxige52** [reply]
求解 求解 大佬救救

**Comment by ddaodan** [comment]
v3.11.0 已修复

**Reply by Caesar0711** [reply]
求教虾，openwrt istoreos怎么更新版本呀？

**Reply by Huanxige52** [reply]
3.11 可以正常使用了吗， ？ 阿里云

**Comment by JoeChen2me** [comment]
经过测试，使用V3版本的Alist挂载阿里云盘分享链接还是可以使用的。拿来看片没问题了。

**Reply by nb-sb** [reply]
这个版本有链接不，我看一下

**Reply by JoeChen2me** [reply]
我写个教程出来 后续发布

**Reply by JoeChen2me** [reply]
写好了[教程](https://peach99cpp.github.io/2023/02/14/Alist%E6%8C%82%E8%BD%BD%E9%98%BF%E9%87%8C%E4%BA%91%E7%9B%98/#more)，发布在我的博客上了，你看看

**Comment by JoeChen2me** [comment]
写了一个[教程](https://peach99cpp.github.io/2023/02/14/Alist%E6%8C%82%E8%BD%BD%E9%98%BF%E9%87%8C%E4%BA%91%E7%9B%98/#more)，发布在我的博客上了。用挂载分享链接的方式实现原画质的访问

**Reply by Huanxige52** [reply]
大佬 怎么联系您？

**Reply by JoeChen2me** [reply]
看见我上面那个蓝色的字吗？点进去就是。我的博客里有我的邮箱

**Comment by 52eraja** [comment]
alist-replit 如何升级呢？

**Reply by Daniel041** [reply]
如果你和我一样是
win版本的话可以用打开github最新的3.11版本下载替换掉原来的exe文件就可以了

**Reply by ywjzzz** [reply]
如何替换呢 之前是replit托管的吧

**Comment by StandGreat** [comment]
+1

**Comment by starsink-d** [comment]
升级到最新版解决所有问题




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年2月16日(星期四) 下午2:18
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [alist-org/alist] Alist挂在阿里云盘出现failed get link: invalid X-Device-Id (Discussion #3381)





 
+1

**Reply by Windlien** [reply]
升级了最新的windows386还是不行 看视频出错了。
GET https://cn-beijing-data.aliyundrive.net/52DhROva%2F964084%2F609d41ce7400f14f9844493b89aca811a691a300%2F609d41ce1acf397c642147ecb9f2e2f8ee081cd5?di=bj29&dr=1793026&f=639869c10fafb5c8277b42b6a0aeb6ba142e48c1&response-content-disposition=attachment%3B%20filename%2A%3DUTF-8%27%27SE01.02.mkv&security-token=CAIS%2BgF1q6Ft5B2yfSjIr5fWPszihJxYzZedbUWCl00hO8xYl4LG1Tz2IHFPeHJrBeAYt%2FoxmW1X5vwSlq5rR4QAXlDfNX3CE37ZqVHPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLukMWr%2Fwq3PIdp2ma447NWQlLnzyCMvvJ9OVDFyN0aKEnH7J%2Bq%2FzxhTPrMnpkSlacGoABJ4dCh4wxtf1DQNg9DQxg1TEcdKufjA81vBn7BFu0RVe20yNwEXP5sKQCtTtFI8eppK6alBIiq%2BTJ88KgPDB7M7pJLsWxY7QkFHR%2Fd0lgyBABMKKvZgxg2xbl2rtxIhcFsnghtTYMD3gOHds1K%2FwX9UEsoOJ1Oz8r9rtq2J1RkDw%3D&u=fb1ed5254f8d4020839860add954273a&x-oss-access-key-id=STS.NTcuvViCyzUvoc3sMt7CtxGm7&x-oss-additional-headers=referer&x-oss-expires=1676548746&x-oss-signature=L5tZUWnR1lsEEDqP%2BM2zlWDHctlhcu0m%2FrqTkms3Vnk%3D&x-oss-signature-version=OSS2 403 (Forbidden)

**Reply by GSANDLY** [reply]
由于阿里云盘 referer 的限制，必须使用移动端 token。 使用桌面 Web 令牌将导致无法下载和预览。 当然，如果你在本地使用或者带宽足够大，你也可以开启代理，让桌面 Web 的 refresh token 正常工作。

https://alist.nn.ci/zh/guide/drivers/aliyundrive.html

**Comment by starsink-d** [comment]
去tg问大佬




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年2月16日(星期四) 下午4:03
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [alist-org/alist] Alist挂在阿里云盘出现failed get link: invalid X-Device-Id (Discussion #3381)





 
升级了最新的windows386还是不行 看视频出错了。
 GET https://cn-beijing-data.aliyundrive.net/52DhROva%2F964084%2F609d41ce7400f14f9844493b89aca811a691a300%2F609d41ce1acf397c642147ecb9f2e2f8ee081cd5?di=bj29&amp;dr=1793026&amp;f=639869c10fafb5c8277b42b6a0aeb6ba142e48c1&amp;response-content-disposition=attachment%3B%20filename%2A%3DUTF-8%27%27SE01.02.mkv&amp;security-token=CAIS%2BgF1q6Ft5B2yfSjIr5fWPszihJxYzZedbUWCl00hO8xYl4LG1Tz2IHFPeHJrBeAYt%2FoxmW1X5vwSlq5rR4QAXlDfNX3CE37ZqVHPWZHInuDox55m4cTXNAr%2BIhr%2F29CoEIedZdjBe%2FCrRknZnytou9XTfimjWFrXWv%2Fgy%2BQQDLItUxK%2FcCBNCfpPOwJms7V6D3bKMuu3OROY6Qi5TmgQ41Uh1jgjtPzkkpfFtkGF1GeXkLFF%2B97DRbG%2FdNRpMZtFVNO44fd7bKKp0lQLukMWr%2Fwq3PIdp2ma447NWQlLnzyCMvvJ9OVDFyN0aKEnH7J%2Bq%2FzxhTPrMnpkSlacGoABJ4dCh4wxtf1DQNg9DQxg1TEcdKufjA81vBn7BFu0RVe20yNwEXP5sKQCtTtFI8eppK6alBIiq%2BTJ88KgPDB7M7pJLsWxY7QkFHR%2Fd0lgyBABMKKvZgxg2xbl2rtxIhcFsnghtTYMD3gOHds1K%2FwX9UEsoOJ1Oz8r9rtq2J1RkDw%3D&amp;u=fb1ed5254f8d4020839860add954273a&amp;x-oss-access-key-id=STS.NTcuvViCyzUvoc3sMt7CtxGm7&amp;x-oss-additional-headers=referer&amp;x-oss-expires=1676548746&amp;x-oss-signature=L5tZUWnR1lsEEDqP%2BM2zlWDHctlhcu0m%2FrqTkms3Vnk%3D&amp;x-oss-signature-version=OSS2 403 (Forbidden)

**Reply by Windlien** [reply]
哦  我看看tg咋用。。。 谢谢

**Comment by kaku1210** [comment]
也是这个问题 更新到最新版就可以用了
curl -fsSL "https://alist.nn.ci/v3.sh" | bash -s update

**Comment by tony12888** [comment]
我也一样
